internal/monitors/collectd/df/df.go

```diff
@@ -49,6 +49,18 @@ type Monitor struct {
 	collectd.MonitorCore
 }
 
+// GetExtraMetrics returns additional metrics to allow through.
+func (c *Config) GetExtraMetrics() []string {
+	var extraMetrics []string
+	if c.ReportInodes {
+		extraMetrics = append(extraMetrics, groupMetricsMap[groupReportInodes]...)
+	}
+	if c.ValuesPercentage {
```

Comment:
I believe there's a a set of metrics that are only emitted when ReportInodes and ValuesPercentage are both true (percent_inodes.*). If that's true then I think we'll just have to hardcode some metrics here (use the generated constants in genmetadata.go though).